src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/UsePropertyInsteadOfCountMethodWhenAvailableTests.cs

```diff
@@ -3,6 +3,7 @@
 using System;
 using System.Collections.Generic;
 using System.Collections.Immutable;
+using System.Text;
```

Comment:
Is this necessary?

Reply:
Nope! I was trying to add a case (with stringbuilder) that would cause my old code to fail but couldn't really make it right.